terraform/nixos-rebuild/variables.tf

```
@@ -19,3 +19,9 @@ variable "target_port" {
   description = "SSH port used to connect to the target_host"
   default     = 22
 }
+
+variable "ssh_private_key" {
+  type        = string
+  description = "Content of private key used to connect to the target_host"
+  default     = ""
```

Otherwise the variable is `set` according to deploy.sh: `test -n ${SSH_KEY+x} && echo yes`

```suggestion
  default     = "-"
```

---

Good catch, thank you !